mods/ra/languages/lua/en.ftl

```diff
@@ -5,423 +5,423 @@ objective-completed = Objective completed
+    the alloy facility is dealt with
+keep-chitzkoi-alive = Keep Chitzkoi Alive
+destroy-alloy-facility = Destroy the alloy facility
+heavy-turret-control-destroyed = Excellent! The heavy turret control center is destroyed;
+    now we can deal with the alloy facility.
 rebuild-chitzkoi = We can rebuild Chitzkoi. We have the technology.
 
 ## survival01
 french-reinforcements-arrive-in = French reinforcements arrive in { $time }
-french-allies-arrived = Our french allies have arrived!
+french-allies-arrived = Our French allies have arrived!
 takeover-reinforcements-kill-remaining-soviet-forces = Take control of French reinforcements and
-    kill all remaining Soviet forces.
+    kill all remaining Soviet forces
 takeover-reinforcements-dismantle-soviet-base = Take control of French reinforcements and
-    dismantle the nearby Soviet base.
+    dismantle the nearby Soviet base
 enforce-position-hold-until-reinforcements = Enforce your position and hold-out the onslaught
-    until reinforcements arrive.
-destroy-two-sames-before-reinforcements = Destroy the two SAM sites before reinforcements
-    arrive.
+    until reinforcements arrive
+destroy-two-sams-before-reinforcements = Destroy the two SAM sites before reinforcements
+    arrive
 soviets-blocking-gps = The Soviets are blocking our GPS. We need to investigate their new technology.
 capture-hold-soviet-airbase-northeast = Capture and hold the Soviet airbase
-    in the northeast.
+    in the northeast
 french-survived-dismantled-soviet-presence = The French forces have survived and dismantled the Soviet presence in the area!
-kill-remaining-soviet-forces = Kill all remaining Soviet forces.
-dismantle-nearby-soviet-base = Dismantle the nearby Soviet base.
+kill-remaining-soviet-forces = Kill all remaining Soviet forces
+dismantle-nearby-soviet-base = Dismantle the nearby Soviet base
 
 ## survival02
-destroy-all-soviet-forces = Destroy all Soviet forces in the area.
+destroy-all-soviet-forces = Destroy all Soviet forces in the area
 soviet-reinforcements-arrive-in = Soviet reinforcements arrive in { $time }
 reinforced-position-initiate-counter-attack = Excellent work Commander! We have reinforced our position enough to initiate a counter-attack.
 incoming-report = Incoming Report
-destroy-remaining-soviet-forces-area = Destroy the remaining Soviet forces in the area.
+destroy-remaining-soviet-forces-area = Destroy the remaining Soviet forces in the area
 soviet-reinforcements-arrived = Soviet reinforcements have arrived!
 repair-bridges-for-reinforcement = Commander! The Soviets destroyed the bridges to disable our reinforcements. Repair them for additional reinforcements.
-repair-two-southern-bridges = Repair the two southern bridges.
-enforce-position-hold-out-onslaught = Enforce your position and hold-out the onslaught.
+repair-two-southern-bridges = Repair the two southern bridges
+enforce-position-hold-out-onslaught = Enforce your position and hold out the onslaught
```

Comment:
```suggestion
repair-two-southern-bridges = Repair the two southern bridges.
enforce-position-hold-out-onslaught = Enforce your position and hold out the onslaught.
```
This is a valid if old-fashioned use of this term "Enforce" as I understand it. "Fortify" works as an alternative if you want one.